packages/grafana-runtime/src/services/LocationService.ts

```diff
@@ -96,7 +96,8 @@ export class HistoryWrapper implements LocationService {
   }
 
   replace(location: H.Path | H.LocationDescriptor, forceRouteReload?: boolean) {
-    const state = forceRouteReload ? { forceRouteReload: true } : undefined;
+    const prevState = (this.history.location.state as any)?.forceRouteReload;
+    const state = forceRouteReload ? { forceRouteReload: prevState ? prevState + 1 : 1 } : undefined;
```

Comment:
I think we can remove forceRouteReload?: boolean parameter from this function, and move this counter increment logic to the two places that need it (as this function already takes in the location it feels unnecessary to have a boolean option that modifies the object that can already be passed in)